doc/install.rst

```diff
@@ -0,0 +1,57 @@
+Installation
+============
+
+This section provide guidelines for installing DataTransferKit and its TPLs.
+
+Install third-party libraries
+-----------------------------
+
+The following third party libraries (TPLs) are used by mfmg:
+
++------------------------+---------+
+| Packages               | Version |
++========================+=========+
+| ARPACK                 | N/A     |
++------------------------+---------+
+| Boost                  | 1.65.1  |
++------------------------+---------+
+| BLAS/LAPACK            | N/A     |
++------------------------+---------+
+| deal.II                | 8.5     |
++------------------------+---------+
+| MPI                    | N/A     |
++------------------------+---------+
+| Trilinos               | 12.X    |
```

Comment:
If deal.ii uses only Epetra, I assume that it works with a bunch earlier versions too?

Reply:
Yeah it should with older versions but I don't remember which one. It won't work with 11.2 for example.